std/meta.d

```diff
@@ -41,7 +41,6 @@ template Arguments(TList...)
 ///
 unittest
 {
-    import std.typetuple;
```

Comment:
Maybe this should say `import std.meta` instead, to be a self-contained example?